### ISSUE TITLE
Parse.Query.first() not working after parse@3.5.0-alpha.4

### ISSUE THREAD

**Issue by dipendras** [parse-community/Parse-SDK-JS#1592]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-JS/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the version *4.5.0* of [Parse Server](https://github.com/parse-community/parse-server/releases) and the latest version of [Parse JS SDK](https://github.com/parse-community/Parse-SDK-JS/releases). 

### Issue Description
Parse.Query.first() is not working after parse@3.5.0-alpha.4

### Steps to reproduce
let Product = Parse.Object.extend('Product'); 
const query = new Parse.Query(Product);
let result = await query.first();

***NB: *Product* is a name of a class.***

### Actual Outcome
throws error
error.code: undefined
error.message: Cannot read properties of undefined (reading 'className')

### Expected Outcome
object of 'Product' class

###  Environment
Using Parse JS SDK: 

Server
- Parse Server version: `4.5.0`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `Back4app`

Client
- Parse JS SDK version: `3.5.0` 
- https://npmcdn.com/parse@3.5.0/dist/parse.min.js

**Comment by mtrezza** [comment]
Could you please fill out the issue template? It contains a section for a code example and asks other info which helps others to investigate this issue.

**Comment by mtrezza** [comment]
> I can reproduce the issue with the version 4.5.0 of [Parse Server](https://github.com/parse-community/parse-server/releases) 

Can you reproduce this issue with the latest alpha version of Parse Server 5?

**Comment by mtrezza** [comment]
See https://github.com/parse-community/Parse-SDK-JS/issues/1593#issuecomment-1304288039

**Comment by dblythy** [comment]
I can replicate this issue.

1. run `npm run build`
2. Create a unit test:

```js
fit('can load server', (done) => {
    new Parse.Object('TestObject').save()
});
```

3. Create this file in `/dist`:

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <title>Parse Functionality Test</title>
  <script src='parse.js'></script>
</head>
<body>
  <div>
    Parse Login
  </div>
  <script>
    (async function() {
      Parse.initialize('integration');
      Parse.serverURL = 'http://localhost:1337/parse';
      const obj = await new Parse.Query('TestObject').first();
      alert(obj.id)
    })();
  </script>
</body>
</html>
```

4. Run `npm run integration` (so a mock Parse Server is running)
5. Open the HTML file in the browser.

With the distributed code as of 3.5.0, there are errors.

It seems that this issue, #1599, #1596, #1593 are all symptoms of the same problem.

In my testing, the code breaks at `3.5.0-alpha.4`. I'm assuming #1553 is the commit that caused this issue, some babel or gulp dependancy is broken off. I'll continue to investigate.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.5.1-alpha.2](https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1-alpha.2)

**Comment by mtrezza** [comment]
Closing via https://github.com/parse-community/Parse-SDK-JS/pull/1600

**Comment by mtrezza** [comment]
This should be fixed with https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1-alpha.2, could you try it out and let us know?

**Comment by dipendras** [comment]
I confirm that the issue is fixed with parse@3.5.1-alpha.2.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.5.1-beta.2](https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.5.1](https://github.com/parse-community/Parse-SDK-JS/releases/tag/3.5.1)
